app/src/main/java/com/poupa/vinylmusicplayer/dialogs/SongDetailDialog.java

```diff
@@ -17,7 +17,7 @@
 import com.poupa.vinylmusicplayer.R;
 import com.poupa.vinylmusicplayer.discog.tagging.MultiValuesTagUtil;
 import com.poupa.vinylmusicplayer.model.Song;
-import com.poupa.vinylmusicplayer.util.AutoDeleteAudioFile;
+import com.poupa.vinylmusicplayer.util.AutoAudioFile;
```

Comment:
Can you please rename this to `AutoCloseAudioFIle`, which is more appropriate imho.

Reply:
Sure